doc/examples/segmentation/plot_thresholding.py

```diff
@@ -0,0 +1,58 @@
+"""
+============
+Thresholding
+============
+
+Thresholding is used to create a binary image from a grayscale image [1]_.
+If you are not familiar with the details of the different algorithms and the
+underlying assumptions, it is often difficult to know which algorithm will give
+the best results. Therefore, Scikit-image includes a function to evaluate
+thresholding algorithms provided by the library. At a glance, you can select
+the best algorithm for you data without a deep understanding of their
+mechanisms.
+
+.. [1] https://en.wikipedia.org/wiki/Thresholding_%28image_processing%29
+
+"""
+import matplotlib
+import matplotlib.pyplot as plt
+
+from skimage import data
+from skimage.filters import thresholding
+
+img = data.page()
+
+# Here, we specify a radius for local thresholding algorithms.
+# If it is not specified, only global algorithms are called.
+fig, ax = thresholding.try_all_threshold(img, radius=20,
+                                         figsize=(10, 8), verbose=False)
+plt.show()
+
+######################################################################
```

Comment:
If you pick one of the thresholding algorithms, why use mean thresholding, which is not that useful (I think)? I would rather go for Otsu here, since it's the most widely used algorithm. What do you think?